src/components/Elements/DataGrid/columnsData.tsx

```diff
@@ -30,35 +43,62 @@ export const AssetsData: IDataProvider = {
     },
     {
       field: 'image',
+      headerName: 'Image',
       width: 150,
       sortable: false,
       disableExport: true,
       filterable: false,
-      renderCell: (params) => (
-        <img
-          alt="Asset"
-          src={params.value}
-          style={{
-            width: '100%',
-            height: undefined,
-          }}
-        />
-      ),
+      renderCell: (params) => {
+        return (
+          <img
+            alt="Asset"
+            src={'http://137.74.158.36:81/storage/' + params.value}
```

Comment:
Adres storage weźmy lepiej z .env. Możemy dodać nową zmienną i uwzględniać ją przy walidacji (validateEnv.js)

Reply:
To w sumie będzie niepotrzebne jak ustaliliśmy że zdjęcia jednak nie będą szły przez z storage